src/Models/ShallowWaterModels/shallow_water_cell_advection_timescale.jl

```diff
@@ -0,0 +1,18 @@
+"Returns the time-scale for advection on a regular grid across a single grid cell 
+ for ShallowWaterModel."
+
+import Oceananigans.Utils: cell_advection_timescale
+
+function shallow_water_cell_advection_timescale(uh, vh, h, grid)
+    uhmax = maximum(abs, uh)
+    vhmax = maximum(abs, vh)
+     hmin = minimum(abs,  h)
+
+    return min(grid.Δx/uhmax*hmin, grid.Δy/vhmax)*hmin
```

Comment:
Something is wrong with the parentheses. Perhaps we want

```suggestion
    return min(grid.Δx / uhmax, grid.Δy / vhmax) * hmin
```

Reply:
Good catch and I have fixed that as well.